src/modules/site-v2/templates/primary/map.html

```diff
@@ -0,0 +1,240 @@
+      <div><input type='checkbox' name='filters' value='tropicalis_layer'><i> C. Tropicalis</i></div>
+    </form>
+  </div>
+   
+    <div class='text-center'>
+        <small><strong>Hover over or click a pin to see information about a <i>C. elegans</i> wild isolate</strong></small>
+    </div>{# /text-center #}
+</div>
+
+{% block script %}
+
+<script type="module">
+
+// Icons
+{% autoescape off %}
+const icon_elegans = L.icon({
+    iconUrl: '{{ ext_asset("img/icons/marker.png") }}',
+    iconSize: [24, 24],
+    iconAnchor: [12, 16],
+    popupAnchor: [0, -15],
+});
+
+const icon_briggsae = L.icon({
+    iconUrl: '{{ ext_asset("img/icons/selected.png") }}',
+    iconSize: [24, 24],
+    iconAnchor: [12, 16],
+    popupAnchor: [0, -15],
+});
+const icon_tropicalis = L.icon({
+    iconUrl: "{{ url_for('static', filename='img/tropicalis-marker.png') }}",
+    iconSize: [24, 24],
+    iconAnchor: [12, 16],
+    popupAnchor: [0, -15],
+});
+
+
+const icon_hover = L.icon({
+    iconUrl: "{{ url_for('static', filename='img/hover-icon.png') }}",
+    iconSize: [24, 24],
+    iconAnchor: [10, 16],
+    popupAnchor: [0, -15],
+});
+{% endautoescape %}
+
+  const MB_ATTR = 'Tiles &copy; Esri &mdash; Source: Esri, DeLorme, NAVTEQ, USGS, Intermap, iPC, NRCAN, METI, TomTom, 2012';
+  const MB_URL = 'https://server.arcgisonline.com/ArcGIS/rest/services/World_Street_Map/MapServer/tile/{z}/{y}/{x}';
+  const OSM_URL = 'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png';
+  const OSM_ATTRIB = '&copy; <a href="https://openstreetmap.org/copyright">OpenStreetMap</a> contributors';
+
+  const southWest = L.latLng(-90, -180);
+  const northEast = L.latLng(90, 180);
+  const bounds = L.latLngBounds(southWest, northEast);
+
+  const map = L.map('map', {minZoom:2, maxBounds: bounds}).setView([20, -100], 2);
+  L.tileLayer(MB_URL, {attribution: MB_ATTR, id: 'mapbox.streets', continuousWorld: false, worldCopyJump: true}).addTo(map);
+  let strain_info = [];
+  const strain_names = [];
+  const elegans = [];
+  const briggsae = [];
+  const tropicalis = [];
+
+
+function naturalCompare(a, b) {
+    const ax = [], bx = [];
+  a = a.strain
+  b = b.strain
+
+  a.replace(/(\d+)|(\D+)/g, function(_, $1, $2) { ax.push([$1 || Infinity, $2 || ""]) });
+  b.replace(/(\d+)|(\D+)/g, function(_, $1, $2) { bx.push([$1 || Infinity, $2 || ""]) });
+  
+  while(ax.length && bx.length) {
+    const an = ax.shift();
+    const bn = bx.shift();
+    const nn = (an[0] - bn[0]) || an[1].localeCompare(bn[1]);
+      if(nn) return nn;
+  }
+
+  return ax.length - bx.length;
+}
+
+function set_marker_color (species) {
+  if (species === 'Caenorhabditis elegans') {
+    return icon_elegans;
+  }
+  if (species === 'Caenorhabditis briggsae') {
+    return icon_briggsae;
+  }
+  if (species === 'Caenorhabditis tropicalis') {
+    return icon_tropicalis;
+  }
+}
+
+const data = {{ strain_listing|tojson|safe }}
+  data.sort(naturalCompare)
+  strain_info = data;
```

Comment:
`strain_info` is declared above, and set to `data` here, but it doesn't appear to be used later in this file. Consider removing the initial declaration `const strain_info = [];`, and on line:137, rename `const data = ` to `const strain_info = `.